hicexplorer/chicViewpoint.py

```diff
@@ -46,34 +45,34 @@ def parse_arguments(args=None):
                                 nargs='+')
 
     parserRequired.add_argument('--range',
-                                help='Defines the region upstream and downstream of a reference point which should be considered in the analysis. Please have in mind to use the same fixate range setting as it was '
-                                'for the background model computation and that distances of the range larger as the fixate range use the background model of those.'
+                                help='Defines the region upstream and downstream of a reference point which should be considered in the analysis. Please remember to use the same fixate range setting as '
+                                'for the background model computation and that distances of the range larger than the fixate range use the background model of those.'
```

Comment:
can you review this line please? Some words seem to be missing, I don't understand what 'those' means.